dpe/runtime_dpe_dpe_fuzz_harness.rs

```diff
@@ -0,0 +1,98 @@
+// Licensed under the Apache-2.0 license
+
+use log::{trace, LevelFilter};
+use simplelog::{Config, WriteLogger};
+use std::fs::OpenOptions;
+
+use crypto::OpensslCrypto;
+use dpe::dpe_instance::DpeEnv;
+use dpe::{commands::Command, response::Response, DpeInstance, Support};
+use platform::{DefaultPlatform, AUTO_INIT_LOCALITY};
+
+// TODO: Review impact.
+const SUPPORT: Support = Support {
+    simulation: true,
+    extend_tci: true,
+    auto_init: true,
+    tagging: true,
+    rotate_context: true,
+    x509: true,
+    csr: true,
+    is_ca: true,
+    is_symmetric: true,
+    internal_info: true,
+    internal_dice: true,
+};
+
+struct SimEnv {
+    platform: DefaultPlatform,
+    crypto: OpensslCrypto,
+}
+
+impl DpeEnv for SimEnv {
+    type Crypto = OpensslCrypto;
+    type Platform = DefaultPlatform;
+
+    fn crypto(&mut self) -> &mut OpensslCrypto {
+        &mut self.crypto
+    }
+
+    fn platform(&mut self) -> &mut DefaultPlatform {
+        &mut self.platform
+    }
+}
+
+// One thing to note: AFL is using persistent mode. If objects aren't dropped, this might double-initialise
+pub fn harness(data: &[u8]) {
+    // NOTE: This is racey
```

Comment:
In what way?

Reply:
Executing multiple workers that all write to the same file will result in an unreadable file. I suppose I meant that there's nothing to prevent them from racing to write individual bytes/lines.

Reply:
Got it. Is it possible to detect from inside the code which worker is executing and make them write to different files?

Can probably work this out in a different PR, but if it isn't too difficult it might be good to just take care of it now.

Reply:
One solution is appending the PID to the filename, the workers' processes persist between calls to the `fuzz_target!`. But does it matter? I used logging for development, but now I think the fuzzer outputs all relevant information. Unless there's some invalid sequence of commands, and we're looking for logic bugs too, I think dropping this is easiest.

Reply:
I'm going to merge this to avoid needing to wait another day. We can drop it in a follow-up CL if you think that's best.